intdataplane/endpoint_mgr.go

```diff
@@ -88,11 +89,12 @@ type endpointManager struct {
 	// hostEndpointsDirty is set to true when host endpoints are updated.
 	hostEndpointsDirty bool
 	// activeHostIfaceToChains maps host interface name to the chains that we've programmed.
-	activeHostIfaceToRawChains  map[string][]*iptables.Chain
-	activeHostIfaceToFiltChains map[string][]*iptables.Chain
+	activeHostIfaceToRawChains    map[string][]*iptables.Chain
+	activeHostIfaceToFiltChains   map[string][]*iptables.Chain
+	activeHostIfaceToMangleChains map[string][]*iptables.Chain
 	// Dispatch chains that we've programmed for host endpoints.
-	activeHostRawDispatchChains  map[string]*iptables.Chain
-	activeHostFiltDispatchChains map[string]*iptables.Chain
+	activeHostRawDispatchChains        map[string]*iptables.Chain
+	activeHostFiltMangleDispatchChains map[string]*iptables.Chain
```

Comment:
Didn't spot that these chains were used in two places at first.  Worth a comment to spell that out?